db/migrate/20180831161349_add_data_to_push_notification.rb

```diff
@@ -0,0 +1,10 @@
+class AddDataToPushNotification < ActiveRecord::Migration
+  def up
+    add_column :push_notifications, :data, :json, null: false, default: '{}'
+    change_column_default(:push_notifications, :data, nil)
```

Comment:
Why this? We already set a default in the previous line, also we have a not null constrain.